resources/views/people/relationship/_relationship.blade.php

```diff
@@ -1,9 +1,25 @@
+@php
```

Comment:
I really don't like using `@php` in Laravel.
In *every* case, it could be done in the Controller instead, and that's the place it has to be moved.

Reply:
can you point me in the right direction and tell me in which file should I place the sort function and where does the ```$relationships``` array comes from, So I can sort them there itself and do the displaying here accordingly

Reply:
The relationships array comes from `resources/views/people/relationship/index.blade.php` file, and the `ContactsController@show` method. Each `@include` use another list of relationships (which is not really optimized code btw...)

You can call the `getLocalizedName` and store it in the relationship object using a `map` method, and reorder the collection with `sortByCollator` macro.